modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -3,132 +3,30 @@
-
-.Query parameters
-|===
-| Parameter | Type
-
-| publisher
-| data
-|===
-
-.Example payload
-[source,json]
-----
-{
-  "id": "56e8a064-dc4b-4428-8085-91c18ea07930",
-  "endpointUri": "http://localhost:8080/api/cloudNotifications/v1/dummy",
-  "uriLocation": "http://localhost:8080/api/cloudNotifications/v1/publishers/56e8a064-dc4b-4428-8085-91c18ea07930",
-  "resource": "/cluster/node/compute-1.example.com/ptp"
- }
-----
-
-.Example oc exec curl command
-[source,terminal]
-----
-$ oc exec -it linuxptp-daemon-5j265 -n openshift-ptp -c cloud-event-proxy -- curl --location --request POST 'http://localhost:8080/api/cloudNotifications/v1/publishers' --header 'Content-Type: application/json' --insecure --data ' {
-  "id": "56e8a064-dc4b-4428-8085-91c18ea07930",
-  "endpointUri": "http://localhost:8080/api/cloudNotifications/v1/dummy",
-  "uriLocation": "http://localhost:8080/api/cloudNotifications/v1/publishers/56e8a064-dc4b-4428-8085-91c18ea07930",
-  "resource": "/cluster/node/compute-1.example.com/ptp"
- }'
-----
-
-=== HTTP method
-
-`GET api/cloudNotifications/v1/publishers`
-
-==== Description
-
-Returns a list of publishers. If publishers exist, a `200 OK` status code is returned along with the list of publishers.
-
-.Example oc exec curl command
-[source,terminal]
-----
-$ oc exec -it linuxptp-daemon-5j265 -n openshift-ptp -c cloud-event-proxy -- curl --location http://localhost:8080/api/cloudNotifications/v1/publishers
-----
-
-.Example return
-[source,json]
-----
-[
- {
-  "id": "56e8a064-dc4b-4428-8085-91c18ea07930",
-  "endpointUri": "http://localhost:8080/api/cloudNotifications/v1/dummy",
-  "uriLocation": "http://localhost:8080/api/cloudNotifications/v1/publishers/56e8a064-dc4b-4428-8085-91c18ea07930",
-  "resource": "/cluster/node/compute-1.example.com/ptp"
- }
-]
-----
-
-== api/cloudNotifications/v1/publishers/<publisher_id>
-
-=== HTTP method
-
-`GET api/cloudNotifications/v1/publishers/<publisher_id>`
-
-==== Description
-
-Returns the publisher with id `<publisher_id>`.
-
-.Query parameters
-|===
-| Parameter | Type
-
-| `<publisher_id>`
-| string
-|===
-
-.Example oc exec curl command
-[source,terminal]
-----
-$ oc exec -it linuxptp-daemon-5j265 -n openshift-ptp -c cloud-event-proxy -- curl --location http://localhost:8080/api/cloudNotifications/v1/publishers/56e8a064-dc4b-4428-8085-91c18ea07930
-----
-
-.Example return
-[source,json]
-----
-{
- "id":"56e8a064-dc4b-4428-8085-91c18ea07930",
- "endpointUri":"http://localhost:8080/api/cloudNotifications/v1/dummy",
- "uriLocation":"http://localhost:8080/api/cloudNotifications/v1/publishers/56e8a064-dc4b-4428-8085-91c18ea07930",
- "resource":"/cluster/node/compute-1.example.com/ptp"
-}
-----
+[NOTE]
+====
+9089 is the default port for the `cloud-event-consumer` container deployed in the application pod. You can configure a different port for your DU application as required.
```

Comment:
```suggestion
`9089` is the default port for the `cloud-event-consumer` container deployed in the application pod. You can configure a different port for your DU application as required.
```

I haven't found an official style entry for this. But the current practice in the docs seems to be formatting port numbers in monospace for body text.

Reply:
Noted.